### ISSUE TITLE
Why the calculation of anomaly_scores of test data need to use its groundtruth label?

### ISSUE THREAD

**Issue by szubing** [liznerski/fcdd#26]
In the trainner.test(), it seems the calculation of anomaly_scores of test data need to use its groundtruth label?  It may be unresonable?

**Comment by liznerski** [comment]
Hey. As you can see [here](https://github.com/liznerski/fcdd/blob/7af3d8eadabee81ab8f7db5dea7f8389ef090213/python/fcdd/training/fcdd.py#L26), the loss function ignores labels as long as the network is in eval mode, and, as you can see [here](https://github.com/liznerski/fcdd/blob/master/python/fcdd/training/bases.py#L236), for testing the network indeed is in eval mode. So, no worries ;)

**Comment by szubing** [comment]
> Hey. As you can see [here](https://github.com/liznerski/fcdd/blob/7af3d8eadabee81ab8f7db5dea7f8389ef090213/python/fcdd/training/fcdd.py#L26), the loss function ignores labels as long as the network is in eval mode, and, as you can see [here](https://github.com/liznerski/fcdd/blob/master/python/fcdd/training/bases.py#L236), for testing the network indeed is in eval mode. So, no worries ;)

My problems is solved. Thank you very much!
